pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -54,7 +54,7 @@
 
 
 class IBMQEmulatorBackend(Backend):
-    """A backend which uses the AerBackend simulator to emulate the behaviour of
+    """A Backend which uses the AerBackend simulator to emulate the behaviour of
```

Comment:
For `AerBackend` and `IBMQBackend` you have moved the docs from the `__init__()` method to the main class. What difference does this make to how they appear, and why not do the same for `IBMQEmulatorBackend`?

Reply:
The sphinx autosummary role takes the class docstring rather than the docstring of the `__init__` method. If I leave the docstring on the `__init__` method then nothing shows up in the table for `AerBackend` and `IBMQbackend`.

Reply:
Why not do the same for `IBMQEmulatorBackend` then?

Reply:
Ah now I see what you mean. The difference was that for `IBMQEmulatorBackend` both the class and `__init__`` method had docstrings. The sphinx autosummary uses the class docstring for the summary.

I have now removed the docstring from the `__init__` method for consistency. There was an additonal statement about the `monitor` parameter in the `__init__` method that wasn't in the class docstring. I added this to the class docstring for completeness.